AtReconstruction/AtPatternModification/AtPatternModification.cxx

```diff
@@ -0,0 +1,10 @@
+#include "AtPatternModification.h"
+
+ClassImp(AtPatternModification);
+
+void AtPatternModification::Reset()
+{
+   fPatternEvent = nullptr;
+   fRawEvent = nullptr;
+   fEvent = nullptr;
+}
```

Comment:
Header just implements a default version of `ModifyPatternEvent` which loops through and calls ModifyTrack on each track in the event.